packages/Molecules/Card/stories.js

```diff
@@ -105,7 +105,7 @@ storiesOf('Molecules/Card', module)
   })))
   .add('Card Content (base)', withReadme(readmeCardContent, () => ({
     components: { RplCardContent },
-    template: `<rpl-card-content :link="link" :image="image" :border="border" :type="type">
+    template: `<rpl-card-content :link="link" :image="image" :border="border" :type="type" :center="center">
   <div v-html="content"></div>
```

Comment:
I feel this should be a `slot` in stead of `prop`, how do you think? @alan-cole ?
But we can ignore it in this PR as it's a big change.

Reply:
I think we'd need further discussion as to how slots would work for this.